schedule/__init__.py

```diff
@@ -363,49 +363,63 @@ def weeks(self):
     @property
     def monday(self):
         if self.interval != 1:
-            raise IntervalError("Use mondays instead of monday")
+            raise IntervalError(
+                "Scheduling jobs every 2 or more weeks is not supported"
```

Comment:
This is already a good improvement to what it was. Resulting in:

```
Traceback (most recent call last):
  File "/home/sijmen/projects/schedule/test.py", line 10, in <module>
    schedule.every(3).monday().do(fooJob)
  File "/home/sijmen/projects/schedule/schedule/__init__.py", line 394, in monday
    raise IntervalError("Scheduling jobs every 2 or more weeks is not supported")
schedule.IntervalError: Scheduling jobs every 2 or more weeks is not supported
```

The line the error is thrown at is `schedule.every(3).monday().do(fooJob)`.

Now I'm thinking that we should mention explicitly that this error is originates from the combination of `.every(>2)` and `monday`. Because from the stacktrace it isn't immediately clear.

I'm thinking: `Scheduling .monday() jobs is only allowed for weekly jobs. Using .monday() on a job scheduled to run every 2 or more weeks is not supported.`

Reply:
Ok, i'll make the changes. But we should do the same for all of the jobs? Monday, Tuesday, Wednesday, and so on?

Reply:
Maybe I am missing the point, who knows ;) To me the syntax `schedule.every(3).monday()` means: `run a job every third monday`. That is the part that is not supported: running a job every 2nd monday. Where does `to run every 2 or more times a week` come from? Do you interpret the `schedule.every(3).monday()` differently?

Reply:
Actually, I was missing the point. Your interpretation is correct, I'll make the changes. Thanks for explaining it!